src/components/Pager/_pager.scss

```diff
@@ -0,0 +1,127 @@
+@import '../../globals/theme/ics-theme';
+@import '../../globals/type/ics-typography';
+@import 'mixins';
+
+// Nested atoms
+@import '../Button/button';
+@import '../Tabs/tabs';
+
+@include exports('pager--ics') {
+  .#{$prefix}--pager {
+    @include reset;
+    width: 100%;
+    border: none;
+    @include type-scale-item(c);
+  }
+
+  .#{$prefix}--pager__center {
+    display: flex;
+    flex: 1;
+    justify-content: center;
+    flex-direction: column;
+    position: absolute;
+    max-width: rem(186px);
+    @include bidi-left(rem(49px));
+  }
+
+  .#{$prefix}--pager__left {
+    position: absolute;
+    @include bidi-left(0);
+    @include bidi-margin-right(15px);
+  }
+
+  .#{$prefix}--pager__right {
+    position: absolute;
+    @include bidi-left(rem(229px));
+  }
+
+  .#{$prefix}--pager__page-item {
+    @include type-scale-item(c, false);
+    @include reset;
+    width: rem(24px);
+    position: relative;
+    display: inline-block;
+    margin: 0 rem(5px);
+    border-radius: 4px;
+    border: 1px solid transparent;
+    background: $field-01;
+
+    &:focus {
+      outline: none;
+      border: 1px solid $brand-01;
+    }
+
+    &:hover {
+      cursor: pointer;
+      color: $ui-01;
+      background: $brand-01;
+    }
+  }
+
+  .#{$prefix}--pager__page-list {
+    display: flex;
+    color: $text-01;
+  }
+
+  .#{$prefix}--pager__button--rewind {
+    .#{$prefix}--pager__tooltip-icon {
+      @include bidi-right(2px);
+    }
+  }
+
+  .#{$prefix}--pager__button {
+    border: 1px solid $field-01;
+    margin: 0 rem(5px);
+    border-radius: 50%;
+    height: rem(24px);
+    width: rem(24px);
+    padding: 0;
+    &:hover {
+      background: $brand-01;
+      border-color: $brand-01;
+      .#{$prefix}--tooltip__icon {
+        fill: $inverse-01;
+        border-radius: 50%;
+      }
+    }
+
+    &:focus {
+      outline: none;
+      box-shadow: none;
+      border: none;
+    }
+  }
+
+  .#{$prefix}--pager__button-icon {
```

Comment:
Should probably add a focus style in here too.

Reply:
added it to line 85